src/lib/server/discord-data.ts

```diff
@@ -0,0 +1,63 @@
+import { REST } from '@discordjs/rest';
+import { Routes } from 'discord-api-types/v10';
+import { DiscordData } from '../schema';
+
+const discordApiToken = process.env.DISCORD_API_TOKEN || '';
```

Comment:
What is that?

Reply:
You need a token to use the discord API

Reply:
So how do you intend to get that token?

Reply:
Oh whoops, I completely forgot to add it to the env in the workflow